olp-cpp-sdk-authentication/include/olp/authentication/TokenResult.h

```diff
@@ -19,7 +19,7 @@
 
 #pragma once
 
-#include <boost/optional.hpp>
+#include <olp/core/porting/optional.hpp>
```

Comment:
looks like it should be in separate block like in the 'authentication/Settings.h' above not sure how strict it is though